script.js

```diff
@@ -0,0 +1,103 @@
+                a.appendChild(document.createTextNode('Feet'));//name options
+                var b = document.createElement('option');
+                b.appendChild(document.createTextNode('Miles'));
+                document.getElementById('convertTo').appendChild(a);//add options
+                document.getElementById('convertTo').appendChild(b);
+                break;
+            case 'Liters':
+                document.getElementById('convertToHolder').className='visible';//visible
+                var a = document.createElement('option');//make options
+                a.appendChild(document.createTextNode('Gallons'));//name options
+                var b = document.createElement('option');
+                b.appendChild(document.createTextNode('Quarts'));
+                document.getElementById('convertTo').appendChild(a);//add options
+                document.getElementById('convertTo').appendChild(b);
+                break;
+            case 'Kilos':
+                document.getElementById('convertToHolder').className='visible';//visible
+                var a = document.createElement('option');//make option
+                a.appendChild(document.createTextNode('Pounds'));//name option
+                document.getElementById('convertTo').appendChild(a);//add option
+                break;
+            case 'Celsius':
+                document.getElementById('convertToHolder').className='visible';//visible
+                var a = document.createElement('option');//make option
+                a.appendChild(document.createTextNode('Ferenheit'));//name option
+                document.getElementById('convertTo').appendChild(a);//add option
+                break;
+
+            default://no select
+                    document.getElementById('convertToHolder').className='hidden';
+        }
+    });
+    this.document.getElementById('action').addEventListener('click',function(e){
+        let hold=document.getElementById('resultHolder');//set result
+        hold.innerHTML='';//clear result
+        let res=document.getElementById('userInput').value;//set result
+
+        let calc=new MetricToImperial(res);
+        switch(document.getElementById('convertTo').value){//calculate result
+            case 'Feet':
+                res=calc.toTwoDecimalPlaces(calc.convertMetersToFeet());
+                break;
+            case 'Miles':
+                res=calc.toTwoDecimalPlaces(calc.convertMetersToMiles());
+                break;
+            case 'Gallons':
+                res=calc.toTwoDecimalPlaces(calc.convertLitersToGallons());
+                break;
+            case 'Quarts':
+                res=calc.toTwoDecimalPlaces(calc.convertLitersToQuarts());
+                break;
+            case 'Pounds':
+                res=calc.toTwoDecimalPlaces(calc.convertKilosToPounds());
+                break;
+            default:
+                res=calc.toTwoDecimalPlaces(calc.convertCelciusToFahrenheit());
```

Comment:
I would have loved to see these result messages be a bit more description. Something like:
x degrees Celsius converts to y degrees Fahrenheit.

The way you have it - there really aren't labels associated with the number to give value to the output. Just a design thing.